Admin/AdminInterface.php

```diff
@@ -266,18 +266,6 @@ public function getParentFieldDescription();
     public function hasParentFieldDescription();
 
     /**
-     * translate a message id.
-     *
-     * @param string $id
-     * @param array  $parameters
-     * @param null   $domain
-     * @param null   $locale
-     *
-     * @return string the translated string
-     */
-    public function trans($id, array $parameters = array(), $domain = null, $locale = null);
-
-    /**
```

Comment:
Okay, I'll revert this.

> Which leads me to the following question : what is the point of this interface?

At the momentan, it's nearly impossible to have a custom implementation. Maybe when adopting this https://github.com/sonata-project/SonataAdminBundle/pull/4091 to the next master, it would make more sence.